docs/most-common-issues.md

```diff
@@ -0,0 +1,14 @@
+Before you do anything else try to remove the project again from your computer and clone it again
+
+```bash
+rm -rf [yourpath]/debugger.html
+git clone https://github.com/devtools-html/debugger.html
+cd debugger.html
+yarn install
+yarn start
+```
+
+Don't forget to ask in our [slack room][slack] or fill an [issue][yarn-issue] if you get stuck.
```

Comment:
It might be a good idea to make our slack more prominent and not mention it at the end of the doc. Let's tell the people we love to help